.devcontainer/devcontainer.json

```diff
@@ -13,5 +13,8 @@
         "DavidAnson.vscode-markdownlint"
       ]
     }
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {}
```

Comment:
nice, didn't know about this. could have used this with act to test Github Workflows as it uses docker:

https://github.com/devcontainers/features/tree/main/src/docker-in-docker

Reply:
Made a post about it in the channel ideas a few weeks ago

Reply:
@ibrahim-kabir do you mean you got act working correctly? if so, this should be part of the doc.

Reply:
Done